picfixPy/vibrance.py

```diff
@@ -1,20 +1,54 @@
-def vibrance(input_img, intensity=5, output_img):
-    
+def vibrance(input_img, intensity=0, display = False, output = ''):
+
     '''
-    Adjust the vibrance of an image, given the intensity specificication
+    Increase or decrease the vibrance of an image, given the intensity specificication.
 
     Input
     ============
-    input_img -- string of path for an image file in .png format
-    intensity -- integer in range [0,10] to specificy intensity of vibrance enhancement applied to the image,
-                 defaults to 5.
-    output_img -- string path for output .png image with adjusted vibrance
+
+    img -- string. Path for an image file in .jpg, .jpeg, .png, .tiff format.
+    intensity -- integer/float in range [-10,10]. Specificies intensity of vibrance applied to image.
+    display -- bool. If True, output image will display
+    output -- string path of output image
 
     Returns
     ============
-    an image file in .png format with specified vibrance in specified output path
+    an image file in .jpg, .jpeg, .png, .tiff format with specified vibrance in specified output path
 
-    Example
```

Comment:
I suggest keeping the example for the documentation.
```suggestion
    Example
    ============
    vibrance("../img/input.png", 5, display = True, "../img/output.png")
```